README.md

```diff
@@ -0,0 +1,66 @@
+The canonical URL for the upstream Spectrum git repository is
+<https://spectrum-os.org/git/spectrum/>.
+
+To try Spectrum OS, you need to build it from source and then install it to an SD card.
+
+> Cross-compiled version of Spectrum OS is for i.MX 8QXP board only.
+
+## Building a Spectrum OS Image
+
+Clone sources of spec and nix-spec (cross-compile branches for i.MX 8QXP):
+
+	$ git clone -b aarch64-imx8-crosscompile https://github.com/tiiuae/nixpkgs-spectrum/
+    $ git clone -b aarch64-imx8-crosscompile https://github.com/tiiuae/spectrum/
+
+Set up the Spectrum binary cache.
```

Comment:
Please add - "Using binary caches speeds up builds significantly."
It helps unfamiliar user to understand why.

Reply:
Changed to:
Set up the Spectrum binary cache to speed up the build process:...